src/pages/index.tsx

```diff
@@ -136,18 +96,18 @@ const Index: FC = () => {
             <Head title="Tasks" />
             <TasksProvider>
                 <div className={classNames.container}>
-                    {!!error && <p>{TASKS_FETCH_ERROR_MESSAGE}</p>}
+                    {isError && <p>{TASKS_FETCH_ERROR_MESSAGE}</p>}
```

Comment:
the optional chaining looks too much nested please fix this in a different PR

Reply:
okay